src/test/java/io/strimzi/kafka/bridge/clients/BasicKafkaClient.java

```diff
@@ -140,73 +148,92 @@ public int sendJsonMessagesPlain(long timeoutMs, String topicName, int messageCo
+                message, 0, timestamp,  false);
+    }
+
+    /**
+     * Send messages to entry-point of the kafka cluster with PLAINTEXT security protocol setting
+     *
+     * @param topicName         topic name where messages are send
+     * @param messageCount      message count
+     * @param message           specific message to send
+     * @param partition         partition count, how many shards/partitions will topic have
      * @param withNullKeyRecord boolean, which allowing sending messages with NULL key
      * @return sent message count
      */
     public int sendJsonMessagesPlain(String topicName, int messageCount, String message, int partition, boolean withNullKeyRecord) {
         return sendJsonMessagesPlain(Duration.ofMinutes(2).toMillis(), topicName, messageCount, List.of(),
-            message, partition, withNullKeyRecord);
+                message, partition, null,  withNullKeyRecord);
     }
 
     /**
      * Send messages to entry-point of the kafka cluster with PLAINTEXT security protocol setting
-     * @param topicName topic name where messages are send
-     * @param messageCount message count
-     * @param headers kafka headers
-     * @param message specific message to send
+     *
+     * @param topicName         topic name where messages are send
+     * @param messageCount      message count
+     * @param headers           kafka headers
+     * @param message           specific message to send
      * @param withNullKeyRecord boolean, which allowing sending messages with NULL key
      * @return sent message count
      */
     public int sendJsonMessagesPlain(String topicName, int messageCount, List<KafkaHeader> headers, String message,
                                      boolean withNullKeyRecord) {
-        return sendJsonMessagesPlain(Duration.ofMinutes(2).toMillis(), topicName, messageCount, headers, message, 0,
-            withNullKeyRecord);
+        return sendJsonMessagesPlain(Duration.ofMinutes(2).toMillis(), topicName, messageCount, headers, message, 0, null, withNullKeyRecord);
     }
 
     /**
      * Send messages to entry-point of the kafka cluster with PLAINTEXT security protocol setting
-     * @param topicName topic name where messages are send
+     *
+     * @param topicName    topic name where messages are send
      * @param messageCount message count
-     * @param message specific message to send
-     * @param partition partition count, how many shards/partitions will topic have
+     * @param message      specific message to send
+     * @param partition    partition count, how many shards/partitions will topic have
      * @return sent message count
      */
     public int sendJsonMessagesPlain(String topicName, int messageCount, String message, int partition) {
         return sendJsonMessagesPlain(Duration.ofMinutes(2).toMillis(), topicName, messageCount, List.of(),
-            message, partition, false);
+                message, partition, null, false);
```

Comment:
no need for this indentation change

Reply:
my IDE's code reformat work :(
I am on it.

Reply:
> my IDE's code reformat work :(

Yeah but before pushing you should pay attention on what it did looking at diff, so you can revert the changes.